README.md

```diff
@@ -130,13 +130,33 @@ developers monitor both channels.
 
 ### News and Updates 📣
 
+- **Feb 2024**: Version 0.14 is released, which deprecates [jj checkout and merge](changelog.md#deprecations),
```

Comment:
nit: Style it as "`jj checkout` and `jj merge`" for clarity?

Reply:
done.